action.yml

```diff
@@ -63,6 +67,10 @@ runs:
         echo "$HOME/.config/guix/current/bin" >> $GITHUB_PATH
       shell: bash
       name: Set PATH
-    - run: guix describe -f channels-sans-intro
+    # Note: substitute newlines to work-around GitHubs single line limitation.
+    # Taken from <https://github.community/t/set-output-truncates-multiline-strings/16852/10>.
+    - run: |
+        echo "::set-output name=channels::$(guix describe -f channels | python3 -c 'import sys; print(sys.stdin.read().replace("\\","\\\\").replace("\"","\\\"").replace("\n","\\n"))')"
```

Comment:
If I understand correctly this quoting magic is mainly required for the `echo` above. Can’t we get away with `| tr '\n' ' '` (to make sure `::set-output` is a single line) and use a heredoc above?